daisy_workflows/image_import/windows/translate.ps1

```diff
@@ -265,25 +268,28 @@ function Install-32bitPackages {
 
 try {
   Write-Output 'Translate: Beginning translate PowerShell script.'
-  $script:outs_dir = Get-MetadataValue -key 'daisy-outs-path'
-  $script:install_packages = Get-MetadataValue -key 'install-gce-packages'
-  $script:sysprep = Get-MetadataValue -key 'sysprep'
-  $script:is_byol = Get-MetadataValue -key 'is_byol'
-  $script:is_x86 = Get-MetadataValue -key 'is_x86'
```

Comment:
Should this be removed from translate Daisy workflows as well?

Reply:
Yep, good catch. I've added that change below.